lib/rubocop/cop/rails/reversible_migration.rb

```diff
@@ -244,14 +244,20 @@ def check_change_table_node(node, block)
         def check_change_table_offense(receiver, node)
           method_name = node.method_name
           return if receiver != node.receiver &&
-                    !IRREVERSIBLE_CHANGE_TABLE_CALLS.include?(method_name)
+                    !IRREVERSIBLE_CHANGE_TABLE_CALLS.include?(method_name) &&
+                    !irreversible_change_default?(node)
 
           add_offense(
             node,
             message: format(MSG, action: "change_table(with #{method_name})")
           )
         end
 
+        def irreversible_change_default?(node)
+          node.method_name == :change_default &&
+            !all_hash_key?(node.arguments.last, :from, :to)
+        end
```

Comment:
It would be better to replace `IRREVERSIBLE_CHANGE_TABLE_CALLS` constant with the following method.

```diff
diff --git a/lib/rubocop/cop/rails/reversible_migration.rb b/lib/rubocop/cop/rails/reversible_migration.rb
index 782b151fd..bf0179c57 100644
--- a/lib/rubocop/cop/rails/reversible_migration.rb
+++ b/lib/rubocop/cop/rails/reversible_migration.rb
@@ -127,9 +127,6 @@ module RuboCop
       # @see https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html
       class ReversibleMigration < Cop
         MSG = '%<action>s is not reversible.'
-        IRREVERSIBLE_CHANGE_TABLE_CALLS = %i[
-          change remove
-        ].freeze

         def_node_matcher :irreversible_schema_statement_call, <<-PATTERN
           (send nil? ${:change_table_comment :execute :remove_belongs_to} ...)
@@ -244,8 +241,7 @@ module RuboCop
         def check_change_table_offense(receiver, node)
           method_name = node.method_name
           return if receiver != node.receiver &&
-                    !IRREVERSIBLE_CHANGE_TABLE_CALLS.include?(method_name) &&
-                    !irreversible_change_default?(node)
+                    reversible_change_table_call?(node)

           add_offense(
             node,
@@ -253,9 +249,15 @@ module RuboCop
           )
         end
-        def irreversible_change_default?(node)
-          node.method_name == :change_default &&
-            !all_hash_key?(node.arguments.last, :from, :to)
+        def reversible_change_table_call?(node)
+          case node.method_name
+          when :change, :remove
+            false
+          when :change_default
+            all_hash_key?(node.arguments.last, :from, :to)
+          else
+            true
+          end
         end

         def within_change_method?(node)
```

Reply:
I updated. Thank you for your review. :heart: